src/utils/string.js

```diff
@@ -1,2 +1,8 @@
+import EmojiConvertor from 'emoji-js';
+
+const emoji = new EmojiConvertor();
```

Comment:
Normally, the converter automatically determines what output to use between `native`, `html`, `css`, `text`, etc.
In a node environment like gatsby build time, it used `html`.

It turns out that we can force the output to be `native` at all times using the `replace_mode` option.

```suggestion
const emoji = new EmojiConvertor();
emoji.replace_mode = "unified";
```